integration_test/cloudbuild-integration-test.yaml

```diff
@@ -387,6 +387,26 @@
   waitFor:
   - run-store-scu
 
+- name: ${_CLOUD_SDK_IMAGE}
+  id: check-store-curl-with-local-backup
+  args:
+    - bash
+    - integration_test/scripts/curl-dcm.sh
+    - "https://healthcare.googleapis.com/${_VERSION}/projects/${_PROJECT}/locations/${_LOCATION}/datasets/${_DATASET}/dicomStores/${_STORE_NAME}/dicomWeb/studies/${_REPLACED_UID}/series/${_REPLACED_UID}/instances/${_REPLACED_UID}"
+    - integration_test/downloaded2.dcm
+  waitFor:
+    - run-store-scu-with-local-backup
+
+- name: ${_CLOUD_SDK_IMAGE}
+  id: check-store-curl-with-gcs-backup
+  args:
+    - bash
+    - integration_test/scripts/curl-dcm.sh
+    - "https://healthcare.googleapis.com/${_VERSION}/projects/${_PROJECT}/locations/${_LOCATION}/datasets/${_DATASET}/dicomStores/${_STORE_NAME}/dicomWeb/studies/${_REPLACED_UID}/series/${_REPLACED_UID}/instances/${_REPLACED_UID}"
```

Comment:
We should use the real study, series, and instance UIDs here instead of REPLACED_UID. The UIDs will not be replaced because the two duplicate adapters do not specify any redaction flags.